frontend/src/TooljetDatabase/Table/index.jsx

```diff
@@ -290,7 +290,8 @@ const Table = ({ collapseSidebar }) => {
         }));
         setCellVal(cellValue);
         cellValue === null ? setNullValue(true) : setNullValue(false);
-        setDefaultValue(false);
+        const findDfaultValue = headerGroups[0].headers[cellClick.cellIndex].column_default === cellVal ? true : false;
```

Comment:
yeah sure, and i think the default state logic is only working for strings only because we are directly comparing cellVal with defaultValue which is coming from backend (which is integer and other data types) and cellVal is directly from input field (which is string data type), that's why its not working for other data types, I missed to check this case, I'll Fix this.